src/Shaders/geometry.fragment.fx

```diff
@@ -46,4 +76,18 @@ void main() {
 
     gl_FragData[VELOCITY_INDEX] = vec4(velocity, 0.0, 1.0);
     #endif
+
+    #ifdef REFLECTIVITY
+        #ifdef HAS_SPECULAR
+            // Specular
+            vec4 reflectivity = texture2D(reflectivitySampler, vReflectivityUV);
+        #elif HAS_REFLECTIVITY
+            // Reflectivity
+            vec4 reflectivity = vec4(texture2D(reflectivitySampler, vReflectivityUV).rgb, 1.0);

```

Comment:
PBR metallic/roughness/etc. not taken into account for now. The renderer simply takes the reflectivity map and doesn't compute any metallic/roughness function at the moment.